src/FailureDetectorConfig.md

```diff
@@ -85,3 +88,65 @@ config.setProperty("hazelcast.heartbeat.phiaccrual.failuredetector.sample.size",
+- `hazelcast.icmp.ttl` (default 0) - The maximum number of hops the packets should go through or 0 for the default
+
+In the above configuration, the Ping detector will attempt 3 pings, one every second, and will wait up-to 500ms for each to complete. If after 3 seconds,
+there was no successful ping, the member will get suspected.
+
+To enforce the [Requirements](#Requirements) the property `hazelcast.icmp.echo.fail.fast.on.startup` can also be set to `true`, in which case, if any of the requirements
+isn't met, Hazelcast will fail to start.
+
+Below a summary table of all possible combinations of the ping failure detector.
+
+| ICMP  	| Parallel 	| Fail-Fast 	| Description                                                                                                                                                                                                                                                                                                     | Linux                                                                                                  	| Windows                       	| macOS                                                                	|
+|-------	|----------	|-----------	|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |--------------------------------------------------------------------------------------------------------	|-------------------------------	|----------------------------------------------------------------------	|
+| false 	| false    	| false     	| Completely disabled                                                                                                                                                                                                         	                                                                                  | N/A                                                                                                    	| N/A                           	| N/A                                                                  	|
+| true  	| false    	| false     	| Legacy ping mode. This works hand-to-hand with the OSI Layer 7 failure detector (see. phi or deadline in sections above). Ping in this mode will only kick in after a period of no hearbeats received, in which case the remote node will be pinged up-to 5 times. If all 5 attempts fail, node gets suspected. | Supported  ICMP Echo if available - Falls back on TCP Echo on port 7                                   	| Supported  TCP Echo on port 7 	|  Supported ICMP Echo if available - Falls back on TCP Echo on port 7 	|
```

Comment:
....mode and will only kick in after a period when there are no hearbeats received, in which case the remote Hazelcast member will be pinged up to 5 times. If all 5 attempts fail, the member gets suspected....